libs/perun/components/src/lib/table-wrapper/table-wrapper.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { MatPaginator, PageEvent } from '@angular/material/paginator';
+import { TABLE_ITEMS_COUNT_OPTIONS } from '@perun-web-apps/perun/utils';
+
+@Component({
+  selector: 'perun-web-apps-table-wrapper',
+  templateUrl: './table-wrapper.component.html',
+  styleUrls: ['./table-wrapper.component.css']
+})
+export class TableWrapperComponent {
+
+  public paginator: MatPaginator;
+  @Input()
+  hideExport = false;
+  @Output()
+  page = new EventEmitter<PageEvent>();
+  @Input()
+  pageSizeOptions = TABLE_ITEMS_COUNT_OPTIONS;
+  @Input()
+  pageSize = 5
```

Comment:
This component could actually handle all the logic when pageSize is changed.
Instead of getting pagesize as input, you can get tableId as input and if the user change the pageSize the component can right here change it because it has all it needs to write it to localhost
Try to think about it. If you this this is too big change we can do it in the next task/in next spring

Reply:
Yes, you are right, I suggest making a new task out of it since it affects a lot of files (lists + their parent components) and I won't be able to do it this sprint.